src/win.js

```diff
@@ -34,16 +34,19 @@ const binPowerShell = "powershell.exe";
  * @returns {string} The escaped argument.
  */
 function escapeArgCmd(arg, { interpolation, quoted }) {
-  let result = arg
-    .replace(/[\0\u0008\u001B\u009B]/gu, "")
-    .replace(/\r?\n|\r/gu, " ");
+  let result = arg.replace(/[\0\u0008\u001B\u009B]/gu, "");
 
   if (interpolation) {
-    result = result.replace(/\^/gu, "^^").replace(/(["&<>|])/gu, "^$1");
+    result = result
+      .replace(/\^/gu, "^^")
+      .replace(/(["&<>|])/gu, "^$1")
+      .replace(/([\t ])/gu, "^$1");
```

Comment:
Based on fuzzing, it appears CMD does not treat escaped tabs or spaces differently from not-escaped ones. That is, tabs are still converted to spaces, and consecutive spaces are still combined into a single space.

From fuzzing it's not clear if this prevents argument splitting.